lib/ssh_scan/constants.rb

```diff
@@ -18,6 +21,6 @@ module Constants
                             "2d63747200000021686d61632d6d64352c686d61632d736861312c686d61632" +
                             "d726970656d6431363000000021686d61632d6d64352c686d61632d73686131" +
                             "2c686d61632d726970656d64313630000000046e6f6e65000000046e6f6e650" +
-                            "00000000000000000000000006e05b3b4").unhexify
+                            "00000000000000000000000006e05b3b4").unhexify)
```

Comment:
The reasoning here was mainly to be more transparent about what we're actually sending so if someone runs into a bug or compatability thing they can tell us something like "... hey I noticed you only include these ciphers, maybe you should add XYZ?"  this I think is better.